server/db/models/Item.js

```diff
@@ -0,0 +1,42 @@
+const Sequelize = require('sequelize');
+const db = require('../database');
+
+const Item = db.define('item', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  description: {
+    type: Sequelize.TEXT
+  },
+  price: {
+    type: Sequelize.DECIMAL(10, 2),
+    allowNull: false
```

Comment:
you could also have a validate: { notEmpty: true } here. every attribute of this model should have allowNull: false and notEmpty: true right?